_docs/versioning.md

```diff
@@ -0,0 +1,184 @@
+---
+
+# Versioning
+
+Versioning documentation is a useful feature that can be achieved using Docsy Jekyll. It allows you to take snapshots of your documentation that may align with a specific version of a product or service, or you may wish to just keep a historical view of the changes that have been made to your documents over time.
+
+{% include alert.html type="info" title="Note: This only allows you to version documents inside the _docs folder, it does not allow you to version any files outside of that, such as Pages, Posts etc." %}
+
+
+## Implementation
+
+The way in which versioning is achieved is by duplicating your whole document directory and placing it inside an 'archive' subdirectory with the name of that version. The following illustrates what the base document root looks like without another version for this Docsy Jekyll site.
+
+```
+document root
+│      
+└─── _data
+└─── _docs
+        └─── extras
+        └─── subfolder
+        example-page.md
+        getting-started.md
+        subfolder.md
+        versioning.md
+```
+To create a version of this documentation we first create a folder with the version name we want to use inside of the 'archive' folder and copy all the files and folders from our root _docs folder into that new folder, we therefore end up with a structure such as below where we moved the older documentation into the folder 'Previous'. Once we have done that we effectively have taken a snapshot of our documents at a point in time and will no longer maintain these. If you look at the structure of this site you will see the 'Previous' folder, which now holds our previous documentation, does not contain the versioning.md markdown file as this is new in the current release.
+
+<pre><code>
+document root
+│      
+└─── _data
+└─── _docs
+        └─── extras
+        └─── subfolder
+        └─── Archive
+              └─── <span style="color:blue;"><b>Previous</b></span>
+                  └─── extras
+                  └─── subfolder
+                  example-page.md
+                  getting-started.md
+                  subfolder.md
+        example-page.md
+        getting-started.md
+        subfolder.md
+        versioning.md
+</code></pre>
+
+You would now continue to add, update or remove markdown files from the main _docs directory in order to work on the current version of your documentation. 
+
+Once the versioning has been correctly configured and you have created a snapshot, you can access those documents by adding the version to the url to access that archived version (e.g. /docs/Archive/Previous/getting-started). 
+
+To enable the users to interact with these versioned documents through the UI there needs to be some updates to the _config.yml file to setup and enable it. The following is a snippet from the _config.yml file specific to the versioning options that are available.
+
+```yml
+version_params:
+  version_menu: "Release"
+  version_dir: Archive
```

Comment:
Perhaps put a note also that this needs to correspond to the folder name too?

Reply:
I did add some additional information about that parameter in the table below that config when I first added it. Is that sufficient or do you think it needs to be more explicit than that?

Reply:
I think it's sufficient!